torch/csrc/autograd/python_engine.cpp

```diff
@@ -137,6 +157,11 @@ PyObject *THPEngine_run_backward(THPEngine *self, PyObject *args, PyObject *kwar
     auto& variable = ((THPVariable*)_variable)->cdata;
     THPUtils_assert(!variable->is_volatile,
         "element %d of variables tuple is volatile", i);
+    // If grad_fn is NULL (as is the case for a leaf node), we instead
+    // interpret the gradient function to be a grad accumulator,
+    // which will accumulate its inputs into the grad property of the
+    // variable.  If inputs/only_inputs is specified, a callback is
+    // used to suppress the operation of this node.
```

Comment:
the callback comment doesn't refer to this part of code, so it's not the best place for it